AtelierTomato.Markov/AtelierTomato.Markov.Generation/Keyword.cs

```diff
@@ -0,0 +1,17 @@
+﻿using AtelierTomato.Markov.Data;
+using Microsoft.Extensions.Options;
+
+namespace AtelierTomato.Markov.Generation
+{
+	public class Keyword(IWordAccess wordAccess, IOptions<KeywordOptions> options)

```

Comment:
I don't really get what it does, please summarize in a /// comment and rename if it enlightens you.
otherwise it looks like something that would derive from Word but this very much isn't, this is more of a service thing